src/map/mob.cpp

```diff
@@ -719,6 +719,12 @@ int32 mob_once_spawn(map_session_data* sd, int16 m, int16 x, int16 y, const char
 
 		mob_spawn(md);
 
+		if (!md->db->title.empty())
+			safestrncpy(md->ud.title, md->db->title.c_str(), NAME_LENGTH);
+		md->ud.group_id = md->db->group_id;
```

Comment:
Those lines are necessary? From my understanding the title and group are defined in `status_calc_mob`. `status_calc_mob(md, SCO_FIRST);` is called in `mob_spawn(md);`

Reply:
Yeah but I think what is actually missing is the clif_name_area call.
Since this is not included in clif_spawn
But that should be called in general and not only here specifically.
And guess the client maybe does not ask for the name of mobs.
So this should probably be added to clif_spawn and clif_getareachar_unit instead.